### PR TITLE
Add my plugins

### PR DESCRIPTION
**Pull request by qwerty287**

### CODE REVIEW

File: docs/plugins/woodpecker-plugins/plugins.json [hunk 1]
```diff
@@ -25,6 +25,18 @@
       "docs": "https://raw.githubusercontent.com/woodpecker-ci/plugin-s3/master/docs.md",
       "verified": true
     },
+    
+    {
+      "name": "Block Git changes",
+      "docs": "https://codeberg.org/qwerty287/woodpecker-block-git-changes/raw/branch/main/docs.md",
+      "verified": true
```
[the remark this answers is not in the document]
Comment (6543) [reply docs/plugins/woodpecker-plugins/plugins.json:32]: well we once said verified is only what does follow the https://github.com/woodpecker-ci/woodpecker/blob/master/CONTRIBUTING.md

with branch protection etc setup ...
... so yes you are a trusted source but not an official plugin

not sure if this field name is bit misleading in this case ...

[the remark this answers is not in the document]
Comment (6543) [reply docs/plugins/woodpecker-plugins/plugins.json:32]: tldr: if repo is `https://codeberg.org/woodpecker-plugins/*` or `https://.github.com/woodpecker-ci/plugin-*`
-> need 2 reviews ...
-> ci do test & lint ...

File: docs/plugins/woodpecker-plugins/plugins.json [hunk 2]
```diff
@@ -25,6 +25,18 @@
...
+    },
+    {
+      "name": "Regex check",
+      "docs": "https://codeberg.org/qwerty287/woodpecker-regex-check/raw/branch/main/docs.md",
+      "verified": true
+    }
+
```
Comment (anbraten) [inline docs/plugins/woodpecker-plugins/plugins.json:39]:
```suggestion
    },
```

File: docs/plugins/woodpecker-plugins/plugins.json [hunk 3]
```diff
@@ -25,6 +25,18 @@
       "docs": "https://raw.githubusercontent.com/woodpecker-ci/plugin-s3/master/docs.md",
       "verified": true
     },
+    
+    {
```
Comment (anbraten) [inline docs/plugins/woodpecker-plugins/plugins.json:29]:
```suggestion
    },
    {
```

**Comment by anbraten** [comment]
@qwerty287 Both your plugins use the same name in the `docs.md` which lets the CI fail.

**Review by anbraten — APPROVED** [review]

**Comment by anbraten** [comment]
Maybe you could add some icons in the long term.

**Pull request merged**